### ISSUE TITLE
💡 [REQUEST] - label native programs like system program, vote, token, incinerator

### ISSUE THREAD

**Issue by jim4067** [helius-labs/xray#174]
### Summary

Label default programs e.g 'system program' or 'vote program'

### Basic Example

1. native programs e.g 1nc1nerator11111111111111111111111111111111 - incinerator should be labelled 

### Drawbacks

none

**Comment by ssamkkim** [comment]
#171 added default program labeling but it is only for log messages. Do you mean on the account page it should show the program name also? I think that would be pretty nice and something worth looking into (especially since we have a default program name function already)

**Comment by jim4067** [comment]
Yes. For the account page. Maybe I was expecting something different. Let me look for it in the code.

**Comment by ssamkkim** [comment]
It's not in the codebase yet, so look into #171 if you want to look into it.
